README.md

```diff
@@ -41,14 +41,14 @@
   </a>
 </p>
 
-Semgrep is a simple and flexible [command-line](#installation) and [online](https://semgrep.live/) tool for static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
+Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.live/).
```

Comment:
I agree with specifying that this is an offline thing.
```suggestion
Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.live/editor).
```

Reply:
thanks @pabloest , good call